src/cascadia/TerminalApp/ActionAndArgs.cpp

```diff
@@ -33,6 +33,7 @@ static constexpr std::string_view ToggleFullscreenKey{ "toggleFullscreen" };
 static constexpr std::string_view SetTabColorKey{ "setTabColor" };
 static constexpr std::string_view OpenTabColorPickerKey{ "openTabColorPicker" };
 static constexpr std::string_view RenameTabKey{ "renameTab" };
+static constexpr std::string_view ExecuteCommandlineKey{ "wt" };
```

Comment:
Yea, I discussed this with @DHowett briefly. There aren't great options here - `execute`? `commandline`? `runCommandline`? `run`? All those felt bad. `wt` felt better because it was kinda like "run `wt` with this commandline"

Reply:
Let's summon the wordsmith @cinnamon-msft to get an expert opinion on the matter 😉

Reply:
I think I like `wt` the best out of those options, since it aligns with the command line arguments.